### PR TITLE
Update `Official tournament support` with multiple minor changes

### PR DESCRIPTION
**Pull request by Albionthegreat**
## Changelog

- Update the minimum size requirement for tournaments and add the minimum size requirement for LAN tournaments.
- Remove conflicting information regarding automated refereeing.
- Update required information to be sent upon tournament conclusion to include links to publicly viewable documents with qualifier results, mp links and mappools.
- Add requirement that all lists of staff are up to date and in sync with one another.
- Update terminology, eliminating the term 'cooldown' from the official support page.

## Checklist

- [x] Approval from the tournament committee.

### CODE REVIEW

File: wiki/Tournaments/Official_support/en.md [hunk 1]
```diff
@@ -96,7 +96,10 @@ Once the tournament has concluded, the tournament organisers will need to submit
 
 - A list of applicable badge recipients (a.k.a. the winners of your tournament).
 - An appropriately designed badge ([see below](#profile-badges)).
-- A link to the publicly viewable catalogue of your tournament's match history.
+- Links to the tournament's wiki page, website and/or publicly viewable spreadsheet(s) with:
+  - The tournament's match history with links to all matches, including qualifiers where applicable.
+  - The tournament's mappools.
+  - The qualifier results — for tournaments that use qualifiers.
```
Comment (hburn7) [inline wiki/Tournaments/Official_support/en.md:102]:
```suggestion
  - The tournament's qualifier results, where applicable.
```

File: wiki/Tournaments/Official_support/en.md [hunk 2]
```diff
@@ -141,9 +144,9 @@ With regards to staff roles, the following rules apply:
...
 
-At the support team's discretion, staff members whose conduct directly results in a tournament losing its official support may not fulfil the same roles in, nor be the host or admin of, another badged tournament for at least 4 months. In the event this cooldown is applied to a user, they will be informed directly through the [tournament team](mailto:tournaments@ppy.sh).
+At the support team's discretion, staff members whose conduct directly results in a tournament losing its official support may not fulfil the same roles in, nor be the host or admin of, another badged tournament for at least 4 months. In the event where a staffing ban is applied to a user, they will be informed directly through the [tournament team](mailto:tournaments@ppy.sh).
```
Comment (hburn7) [inline wiki/Tournaments/Official_support/en.md:147]:
```suggestion
At the support team's discretion, staff members whose conduct directly results in a tournament losing its official support may not fulfil the same roles in, nor be the host or admin of, another officially-supported tournament for at least 4 months. In the event a staffing ban is applied to a user, they will be informed directly through the [tournament team](mailto:tournaments@ppy.sh).
```

**Review by hburn7 — CHANGES_REQUESTED** [review]
semantics

**Comment by hburn7** [comment]
These changes have been approved by the Tournament Committee.

**Review by Hiviexd — DISMISSED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**